README.md

```diff
@@ -467,12 +473,15 @@ import com.kodypay.grpc.pay.v1.*;
 PaymentClient paymentClient = new PaymentClient(HOSTNAME, APIKEY);
 
 String storeId = "UUID of assigned store";
-String orderId = "UUID of assigned store";
-String paymentReference = "payment reference";
+String orderId = "UUID of order generated by Kody";
+String paymentReference = "UUID generated for the request with 'pay_' appended: pay_UUID";
```

Comment:
It doesn't _have_ to start with `pay_` prefix I think

Reply:
Also I don't think the `orderId` and `paymentReference` need to be UUIDs

Reply:
I checked kp-core and orderId does not "Need" to be UUID apparently ! 

I will change the wording for both with what you provided

"a unique reference for the payment, sent from the client and returned by the server"